client/src/ingredients/prose.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+export function Prose({ section }) {
+  return <div dangerouslySetInnerHTML={{ __html: section.content }} />;
+}
+
+export function ProseWithHeading({ id, section }) {
+  if (!id) {

```

Comment:
By looking at document.js I don't believe this check is needed anymore?

```
if (!section.value.id || PROSE_NO_HEADING.includes(section.value.id)) {
  return <Prose key={section.value.id} section={section.value} />;
} else {
  return (
    <ProseWithHeading
      key={section.value.id}
      id={section.value.id}
      section={section.value}
    />
  );
}
```

Reply:
Also noticed that the key for `Prose` would be set to undefined if `section.value.id` does not exist.

Reply:
I should've been reading what they are doing when moving them.

Alright, I've changed these two! Thanks for the feedback

Reply:
Thanks @kenrick95! I'd wait for @peterbe to make the call on this but..it looks like we use `ingredient-name${i}` for most ingredients but not all. Wondering if we should just make that the go-to for keys for ingredients across the board.